explore_data/nbconverted/0_explore_input_data.py

```diff
@@ -0,0 +1,180 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import sys
+import glob
+import pandas as pd
+import numpy as np
+import random
+import seaborn as sns
+import umap
+from keras.models import load_model
+from sklearn.decomposition import PCA
+import pickle
+
+from plotnine import (ggplot,
+                      labs,  
+                      geom_line, 
+                      geom_point,
+                      geom_errorbar,
+                      aes, 
+                      ggsave, 
+                      theme_bw,
+                      theme,
+                      xlim,
+                      ylim,
+                      facet_wrap,
+                      scale_color_manual,
+                      guides, 
+                      guide_legend,
+                      element_blank,
+                      element_text,
+                      element_rect,
+                      element_line,
+                      coords)
+
+from ponyo import utils
+
+from numpy.random import seed
+randomState = 123
```

Comment:
Not a big deal, but `randomState` should be `random_state` to keep everything in snake_case